packages/webview_flutter/lib/request.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Represents a request made by the [WebView] widget.
+///
+/// See also:
+/// * [WebView.initialRequest] for how this gets used.
+class Request {
+  /// Constructs a [Request] object with the given [url]
+  /// and [headers].
+  const Request({
+    this.url,
+    this.headers,
```

Comment:
Consider initializing this to an empty mutable Map if not provided. Compare to package:http

https://github.com/dart-lang/http/blob/master/lib/src/base_request.dart#L85